pkg/server/container_execsync.go

```diff
@@ -55,6 +56,22 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 		return nil, fmt.Errorf("container %q is in %s state", id, criContainerStateToString(meta.State()))
 	}
 
+	// Get exec process spec.
+	cntrResp, err := c.containerService.Get(ctx, &containers.GetContainerRequest{ID: id})
+	if err != nil {
+		return nil, fmt.Errorf("failed to get container %q from containerd: %v", id, err)
```

Comment:
right, just trying to be more specific in the error message.. get container doesn't say much. The response includes the container struct that has the spec in it. Was thinking more detail might help down the road.